lib/engine/game/g_1807/game.rb

```diff
@@ -105,12 +105,14 @@ def event_minors_batch3!
         end
 
         def event_u1_available!
-          u1 = @future_companies.delete { |company| company.sym == 'U1' }
+          u1 = @future_companies.find { |company| company.sym == 'U1' }
+          @future_companies.delete(u1)
           @companies << u1
```

Comment:
you can use reject! to do this in one line

Reply:
> you can use reject! to do this in one line

But I need to grab hold of the U1 company object, to add it to the `@companies` array after it has been removed from `@future_companies`. `reject!` doesn't return the item(s) removed from the array, so I'd still need to use something like `find`.